docs/editors/overview.md

```diff
@@ -418,6 +427,12 @@ Language Server Protocol (LSP). These extensions are not necessary for Metals to
 function but they improve the user experience. To learn more about Metals
 extensions, see [integrating a new editor](../integrations/new-editor.md).
 
+## Implicit decorations 
+
+**✅**: Additional information inferred from the code can be show within the code using additional virtual text.
+
+**Shown in hover**: Additional information inferred from the code can be show when hovering over a specific line. That hover only shows the current line.
```

Comment:
```suggestion
**Shown in hover**: Additional information inferred from the code can be show when hovering over a symbol. That hover only shows information for the current line the symbol is one.
```
Not sure if you can actually just hover anywhere on the in other letters to trigger this or if it actually has to be on something. 🤔 take it or leave it.

Reply:
This is anywhere on the line, since it seemed natural as we show the whole line. I will leave it with `line`

Reply:
👍